docs/cow-protocol/tutorials/cow-amm-deployer.mdx

```diff
@@ -2,82 +2,43 @@
 sidebar_position: 6
 ---
 
-# CoW AMM Deployer
-The [CoW AMM Deployer](https://app.safe.global/share/safe-app?appUrl=https%3A%2F%2Fdeploy-cow-amm.bleu.fi&chain=et) is a user-friendly Safe App that simplifies the process of deploying and managing [CoW AMMs](https://cow.fi/cow-amm) (Automated Market Makers) from Safe wallets. It enables users without programming skills to take advantage of the benefits of CoW AMM. 
+# Initial notes
 
-The app was developed by [bleu](https://github.com/bleu-fi) with sponsorship from the [CoW Grants DAO](https://grants.cow.fi/).
+The proposed transactions and interactions, can be executed through Safe's transaction builder, etherscan "Write Contract" or from other UI mechanisms or directly from console.
```

Comment:
Mention "Safe's transaction builder" is somewhat irrelevant and ties the documentation loosely to Safe, which is not necessary as the Balancer implementation is usable by _any_ type of account, smart contract or not.

Additionally, I'd recommend to use more firm language (it is a tutorial after all, so some concrete steps are expected).